osuosl/iam
# Kennric/client rspec

fixes issue #23

## Changes in this PR.

- [X] Add simple client routes with skeleton templates
- [X] Tests simple client routes
- [X] Adds FactoryGirl factory for clients

1. Run rake spec
2. See tests pass
3. Run ruby app.rb
4. See application run/exit without error (a run! directive needs to be added to run the dev server)

@osuosl/devs

This should be merged after #40, as it builds on that infrastructure
